File: java/src/main/java/mx/iteso/adapter/Image.java
package mx.iteso.adapter;

/**
 * new image.
 */
public class Image {
    /**
     * image url.
     */
    private String url;

    /**
     *
     * @param newUrl .
     */
    public Image(final String newUrl) {
        this.url = newUrl;
    }

    /**
     *
     * @return ulr of image.
     */
    public String getUrl() {
        return this.url;
    }
}

File: csharp/AdapterPractice/AdapterPractice/Adapter/Image.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AdapterPractice.Adapter
{
    public class Image
    {
      
        private String url;

      
        public Image(String newUrl)
        {
            this.url = newUrl;
        }

        public String getUrl()
        {
            return this.url;
        }
    }
}
